### ISSUE TITLE
Field appears empty after adding existing

### ISSUE THREAD

**Issue by davejtoews** [silvershop/silverstripe-hasonefield#41]
Using SS 4.4.1 and v3.0.3 of this plug-in.

When clicking "Link Existing" after finding a record with the autocomplete the field is left empty appearing to have no content entered until the page is reloaded.

The javascript console displays the following error

```
[Error] TypeError: this.getGridField is not a function. (In 'this.getGridField()', 'this.getGridField' is undefined)
	onclick (bundle.js:1:1132)
	i (vendor.js:1:2131704)
	(anonymous function) (vendor.js:1:2137528)
	dispatch (Anonymous Script 1 (line 3332))
	eventHandle (Anonymous Script 1 (line 2941))
```

**Comment by MasonD** [comment]
(sorry about the spam there, didn't realise that force pushes would cause this and I kept noticing formatting changes that accidentally running the file through prettier caused)

This was caused by entwine namespacing issues, which I fixed in #44

**Comment by davejtoews** [comment]
@MasonD Is there likely to be a new release in the near future, or should I just pull-in the master branch for now if I want to take advantage of this fix?
